backends/gdx-backend-headless/src/com/badlogic/gdx/backends/headless/mock/graphics/MockGraphics.java

```diff
@@ -191,9 +192,17 @@ public void setVSync(boolean vsync) {
 
 	}
 
+	/** Sets the target framerate for the application. Use 0 to never sleep;  negative to not call the render method at all.
+	 *  Default is 60.
+	 *
+	 * @param fps fps */
 	@Override
 	public void setForegroundFPS (int fps) {
+		this.targetRenderInterval = (long) (fps == 0 ? (fps == 0 ? 0 : -1) : (1F / fps * 1000000000F));
```

Comment:
`: -1` is unreachable.

Last equation should be `((1F / fps) * 1000000000F)` for better accuracy